README.md

```diff
@@ -37,24 +37,29 @@ Configuration is held in json format, `config.json` is used by default and has s
 
 ```
 $ ./price-alert.py --help
-usage: price-alert.py [-h] [-c CONFIG] [-t POLL_INTERVAL]
-
-optional arguments:
-  -h, --help            show this help message and exit
-  -c CONFIG, --config CONFIG
-                        Configuration file path
-  -t POLL_INTERVAL, --poll-interval POLL_INTERVAL
-                        Time in seconds between checks
+Usage: price-alert.py [OPTIONS]
+
+Options:
+  -t, --pool-interval INTEGER  Time in seconds between checks
+  -d, --debug BOOLEAN          If true set debug level to DEBUG
+  --base-url TEXT              base url to crawl.
+  --xpath-selector TEXT        xpath selector to crawl.
+  -i, --items LIST             List of items to crawl
```

Comment:
I'm not sure I like the syntax for passing items via arguments. I have a PR for adding environment variable support, but that's basically the reason I left it to hang :)

Reply:
I don't like it either and with decouple you can load it from envs 😌

Reply:
Yeah that's what blocked me last time, this requires a rework...